terraform/aks/terraform.tf

```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = "1.5.4"
+  required_version = "1.5.5"
```

Comment:
I think if we were to bump this to 1.5.5, we have to do it everywhere now (there are quite a few other places)